app/views/users/index.html.erb

```diff
@@ -1,2 +1,10 @@
-<h1>Users#index</h1>
-<p>This is where the users list will be shown</p>
+<h1>User list</h1>
+<ul>
+  <% @users.each do |user|%>
+    <li>
+      <img src="<% user.photo %>" alt="user photo">
+      <h2><%= link_to user.name, user %></h2>
+      <span>Number of posts: <% user.posts_counter %></span>
+    </li>
+  <% end %>
+</ul>
```

Comment:
- [OPTIONAL] Please, let's follow the provided wireframe to achieve the desired layout of this section. This is how the elements in this section should be aligned:

  ![image](https://user-images.githubusercontent.com/45002654/150009199-419e89c3-d025-4d92-88f6-da826d38db22.png)
  
  This is how your app displays this section:
  
  ![image](https://user-images.githubusercontent.com/45002654/150009439-4be02f27-5e50-4da4-ad58-1b643fee0d1e.png)
  
  As evident above, the image, the name of the user should be displayed in a horizontal alignment.
  
  The problem is using `h2` which is a block-level element and thus every other element is forced to be in its own line.

Reply:
Thanks for the review, I'll do the entire design on Friday, as I'm struggling a bit with Rails and am focusing all of my current time on it